.github/workflows/publish.yml

```diff
@@ -381,10 +377,6 @@ jobs:
       release_url: ${{ steps.release.outputs.url }}
 
     steps:
-      - name: Set tag
-        run: |
-          echo "TAG=$(git tag --points-at ${{github.sha}})" >> $GITHUB_ENV
```

Comment:
instead of using the `tag` we use the release name that was generated in an earlier step